src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -610,7 +610,7 @@ export function getAppointmentSlots(startDate, endDate, forceFetch = false) {
           // for the correct day.
           .map(slot => {
             if (featureVAOSServiceVAAppointments) {
-              let time = moment(slot.start);
+              let time = moment(slot.start).format('YYYY-MM-DDTHH:mm:ss');
```

Comment:
Looks like the 'if' statement is no longer needed. I would just remove it.

Reply:
When paging availableSlots would not have an offset or end with Z but mappedSlots do have an offset and a Z at the end and are converted with moment.tz() wouldn't the if statement still be needed?

Reply:
Not sure but it doesn't hurt anything to leave it as is.